packages/ubuntu_wizard/lib/src/widgets/validated_form_field.dart

```diff
@@ -56,6 +56,15 @@ class ValidatedFormField extends StatefulWidget {
   /// Enables or disables TextField (defaults to true)
   final bool enabled;
 
+  /// An optional custom icon for the show text button
+  final IconData? showTextIcon;
+
+  /// An optional custom icon for the show text button
+  final IconData? hideTextIcon;
+
+  /// Used when the show text button is pressed
+  final VoidCallback? onShowText;
+
```

Comment:
For a generic widget such as `ValidatedFormField`, it would be nicer to allow any widget by exposing `InputDecoration.prefix`. The logic for showing and hiding the password should be in `_PasswordFormField`.